cmd/podman/containers/ps.go

```diff
@@ -309,7 +309,13 @@ func (l psReporter) Status() string {
 
 // Command returns the container command in string format
 func (l psReporter) Command() string {
-	return strings.Join(l.ListContainer.Command, " ")
+	command := strings.Join(l.ListContainer.Command, " ")
+	if !noTrunc {
+		if len(command) > 17 {
+			return command[0:17] + "..."
```

Comment:
I think you have an off-by-1 here, should be 0:16

Reply:
I looked at cmd/podman/shared/container.go in v1.9.3 branch and there it is 17.